test/api/activity_types_api_test.rb

```diff
@@ -50,4 +50,50 @@ def test_put_activity_types
     first_activity_type = ActivityType.first
     assert_json_matches_model(last_response_body, first_activity_type, response_keys)
   end
+
+  def test_post_activity_types_cannot_auth
```

Comment:
Please change the test name into `test_student_cannot_post_activity_type` for clearer description